src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -73,11 +73,13 @@ const TableRowAccount = forwardRef<TableRowAccountProps, 'div'>(({ asset, accoun
   const pubkey = useMemo(() => fromAccountId(accountId).account, [accountId])
   const isUtxoAccount = useMemo(() => isUtxoAccountId(accountId), [accountId])
 
-  const { data: account, isFetching: isAccountFetching } = useQuery({
+  const { data: account } = useQuery({
     ...accountManagement.getAccount(accountId),
-    staleTime: GET_ACCOUNT_STALE_TIME,
+    staleTime: Infinity,
     // Never garbage collect me, I'm a special snowflake
     gcTime: Infinity,
+    // Yes, we do refetch on mount despite having an Infinity stale time. Stale then fresh FTW.
```

Comment:
And in effect, what we're doing is exactly the same as `portfolio.endpoints.getAccount()` below, just a different way